test/test_helper.rb

```diff
@@ -2,6 +2,8 @@
 require 'minitest/autorun'
 require 'active_model_serializers'
 require 'fixtures/poro'
+require 'rails-controller-testing'
```

Comment:
this can always be required now?

Reply:
yes, previously this was automatically required pre rails 5. now its need to be required when needed for >= rails 5.

Reply:
gotcha.  well, If these changes for you, I'm ok to merge them.

Reply:
seems travis ci is dead and this branch wasn't running a CI. I ported some stuff over in https://github.com/rails-api/active_model_serializers/pull/2447 if you're interested in helping out (happy to let you take it over) and figure out why it's failing and if we care

Reply:
I wouldn't mind taking a look when I have time. Would definitely love to rewrite the lookup to something similar to v0.10 as well. However it does feel like i'm the only one in the world right now still on v0.9 haha!

Just curious, will there still be a release once this gets merged? Or do i have to reference the stable branch for all latest patches?

Reply:
appreciate your prompt response! looks good to me as well! I'll help to look into CI when possible.